lib/ansible/modules/packaging/os/package.py

```diff
@@ -19,7 +19,7 @@
 module: package
 version_added: 2.0
 author:
-    - Ansible Inc
+    - Ansible Core Team
```

Comment:
Maybe we could default this to @ansible ?

Reply:
Everywhere else we use Ansible Core Team, and in the Bot, this was the odd one

https://github.com/ansible/ansibullbot/blob/master/ansibullbot/utils/extractors.py#L522